ruby/agoo/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM ruby:2.5.3
+
+WORKDIR /usr/src/app
+
+COPY Gemfile app.rb ./
+
+RUN bundle install
+
+CMD nproc=256 ruby app.rb
```

Comment:
sorry, mis-understanding
use 
~~~dockerfile
CMD AGOO_WORKER_COUNT=$(nproc) ruby app.rb
~~~
or
~~~dockerfile
ENV AGOO_WORKER_COUNT $(nproc)
CMD ruby app.rb
~~~

`nproc` (unix utility) will determine the number of cores to use

Reply:
I'm not sure it is a good idea to tie nproc to the number of workers but I will make the change tomorrow. Too late here in Tokyo. Just visiting. :-)

Reply:
Ok, back at it for a bit. Worker count now uses nproc.